### DISCUSSION TITLE
Eclipse Theia v1.55.0

### DISCUSSION THREAD

**Discussion by jfaltermeier** [eclipse-theia/theia#14379] [Release Announcements] [answered]
Hi everyone 👋,

The Eclipse Theia v1.55.0 release is scheduled for later today!
We will inform you once the release starts and completes.

**Comment by jfaltermeier** [comment]
The release will start now, I’ll post an update when it has completed.

**Comment by jfaltermeier** [comment, marked as answer]
The release has completed, thank you to everyone that participated and contributed!

**Reply by scharf** [reply]
I am a bit confused about what this means: in https://theia-ide.org/releases/ it looks like 1.55 it will be released later this year:

> [Community Release Announcement](https://github.com/eclipse-theia/theia/discussions/14379): December 12th, 2024

**Reply by JonasHelming** [reply]
We have monthly releases and community releases, see: https://eclipsesource.com/blogs/2022/08/16/introducing-the-eclipse-theia-community-release/

**Comment by JonasHelming** [comment]
[News & Noteworthy](https://eclipsesource.com/blogs/2024/10/09/eclipse-theia-1-54-release-news-and-noteworthy/)
